src/components/Calendar/CalendarEvents.tsx

```
@@ -0,0 +1,282 @@
+																		<span className='text-lightBlue dark:text-icon-dark-inactive'>Proposer:</span>
+																		<Address
+																			address={event?.proposer}
+																			displayInline
+																			iconSize={16}
+																			addressClassName='items-center font-normal'
+																			className='items-center font-normal'
+																			isTruncateUsername={false}
+																		/>
+																	</div>
+																)}
+																{event?.trackNo && !isNaN(event?.trackNo) && (
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)